src/ralph/backends/storage/base.py

```diff
@@ -23,3 +23,7 @@ def read(self, name, chunk_size=4096):
     @abstractmethod
     def write(self, name, chunk_size=4096, overwrite=False):
         """Write content to the `name` target"""
+
+    def fetched_from_backend(self, entry):
```

Comment:
Good point! I was hesitating about that because this utility would depend on both `HistoryMixin` (self.history) and `BaseStorage` (self.name).
For now I put it in the `HistoryMixin` and adding the backend name argument `get_command_history(self, backend_name, command)`